src/store/selectors/solanaWallet.ts

```diff
@@ -64,7 +67,8 @@ export const accountsArray = createSelector(
       if (exchangeAssets[account.programId.toString()]) {
         acc.push({
           ...account,
-          symbol: exchangeAssets[account.programId.toString()].symbol
+          symbol: exchangeAssets[account.programId.toString()].symbol,
+          usdValue: exchangeAssets[account.programId.toString()].price.mul(account.balance).div(new BN(100000000))
```

Comment:
Here you should use Oracle_offset to adjust price

Reply:
done

Reply:
uprzedzając pytania: mnożenie przez 10 ** 4 jest potrzebne, by nie zaginęło parę cyfr po dzieleniach i dokładność nie była zbyt mała